tests/by-util/test_fmt.rs

```diff
@@ -159,3 +185,87 @@ fn test_fmt_set_goal_not_contain_width() {
             .stdout_is("this is a file with one word per line\n");
     }
 }
+
+#[test]
+fn split_does_not_reflow() {
+    for arg in ["-s", "-ss", "--split-only"] {
+        let (at, mut ucmd) = at_and_ucmd!();
+        ucmd.arg("one-word-per-line.txt")
+            .arg(arg)
+            .succeeds()
+            .stdout_is(at.read("one-word-per-line.txt"));
```

Comment:
You can simplify this by using `stdout_is_fixture("one-word-per-line.txt");`. This also applies to the tests below.